tests/integration_tests/conftest.py

```diff
@@ -76,6 +76,17 @@ def disable_subp_usage(request):
 
 @pytest.fixture(scope="session")
 def session_cloud() -> Generator[IntegrationCloud, None, None]:
+    try:
+        cloud = _session_cloud()
+        yield cloud
```

Comment:
Nit: These next two lines can come after the except, right? Otherwise cleanup may never run.

Reply:
Is it possible for `yield cloud` to throw an exception? If not, then I wouldn't expect `cloud` to be a bound variable.